<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemViewer</class>
 <widget class="QWidget" name="ItemViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Item Viewer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="backButton">
       <property name="toolTip">
        <string>Previous</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="shortcut">
        <string>Meta+P</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="playButton">
       <property name="toolTip">
        <string>Play</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="forwardButton">
       <property name="toolTip">
        <string>Next</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="shortcut">
        <string>Meta+N</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="orderGroupBox">
       <property name="title">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="originalRadioButton">
          <property name="toolTip">
           <string>As found in file</string>
          </property>
          <property name="text">
           <string>File Order</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="sortedRadioButton">
          <property name="toolTip">
           <string>Match current sort order</string>
          </property>
          <property name="text">
           <string>Sorted</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="randomRadioButton">
          <property name="toolTip">
           <string>Random order</string>
          </property>
          <property name="text">
           <string>Random</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>File Row:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ItemSpinBox" name="rowSpinBox">
       <property name="toolTip">
        <string>Row Number (Original)</string>
       </property>
       <property name="maximum">
        <number>9999999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="flagCheckBox">
       <property name="text">
        <string>Flag</string>
       </property>
       <property name="shortcut">
        <string>Meta+M</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTableView" name="tableView">
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>357</width>
         <height>384</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="ImageView" name="imageView">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
  <customwidget>
   <class>ItemSpinBox</class>
   <extends>QSpinBox</extends>
   <header>.itemSpinBox</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
